tests/AutoReview/FixerTest.php

```diff
@@ -58,7 +58,9 @@ public function testFixerDefinitions(FixerInterface $fixer)
         $fixerIsConfigurable = $fixer instanceof ConfigurationDefinitionFixerInterface;
 
         $this->assertRegExp('/^[A-Z`].*\.$/', $definition->getSummary(), sprintf('[%s] Description must start with capital letter or a ` and end with dot.', $fixerName));
-        $this->assertPhpUnitCasing($definition->getSummary(), sprintf('[%s] `PHPUnit` must be in correct casing in description.', $fixerName));
+        $this->assertNotContains('phpdocs', $definition->getSummary(), sprintf('[%s] `PHPDoc` must not be in the plural in description.', $fixerName), true);
```

Comment:
is it case sensitive or not?
if it is, then it's not enough

Reply:
[4th](https://github.com/sebastianbergmann/phpunit/blob/7.1.5/src/Framework/Assert.php#L250) parameter causes it's not.